webapp/src/components/sidebar_right/index.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {bindActionCreators} from 'redux';
+
+import {getYourPrDetails, getReviewDetails} from '../../actions';
+import {id as pluginId} from '../../manifest';
+
+import SidebarRight from './sidebar_right.tsx';
+
+function mapPrsToDetails(prs, details) {
+    if (!prs) {
```

Comment:
it would be nice to also check here for the length property to make sure it is an array of values so map wont break if trying to run over a different data structure.